src/lib/src/service/execution/test.execution.service.spec.ts

```diff
@@ -8,14 +8,14 @@ import { Injector, ReflectiveInjector } from '@angular/core';
 import { inject } from '@angular/core/testing';
 import { TestBed } from '@angular/core/testing';
 import { fakeAsync } from '@angular/core/testing';
-import { HTTP_STATUS_CREATED } from '../../component/navigation/navigation.component.test.setup';
+import { HTTP_STATUS_CREATED, HTTP_STATUS_OK } from '../../component/navigation/navigation.component.test.setup';
 
 describe('TestExecutionService', () => {
   let serviceConfig: TestExecutionServiceConfig;
 
   beforeEach(() => {
     serviceConfig = new TestExecutionServiceConfig();
-    serviceConfig.testExecutionServiceUrl = 'http://localhost:9080/tests/execute';
+    serviceConfig.testExecutionServiceUrl = 'http://localhost:9080/tests';
```

Comment:
Well, arguably, both _commencing_ the execution as well as monitoring test execution _status_ is concerned with test _execution_. I do agree, though, that we might want to drop the "execution" part for reasons of clarity.

This would indeed by a very far-reaching renaming. My IDE's refactoring facilities, while quite good, were not sufficient to rename everything one would like to be renamed, at all. In the process of trying to do this renaming, I also managed to mangle everything up by jumping from a feature branch to the master and back again… 

Anyways, I suggest we do this separately, after this PR is through.